data-model.json

```diff
@@ -0,0 +1,149 @@
+{   
+    "name": "Name1 Name2 County",  
+    "update_date": "yyyy/mm/dd ",
+    "update_time": "hh:mm [am/pm]",
+    "source_url": "POINT TO LANDING PAGES, IN CASE ENDPOINTS CHANGE", 
+    "meta_from_source" : "STORE IMPORTANT NOTES FROM SOURCE HERE",
+    "meta_from_baypd": "STORE IMPORTANT NOTES ABOUT OUR METHODS HERE",
+    "series": {
+        "cases": [
+            { "date": "yyyy/mm/dd", "cases": -1, "cumul_cases": -1},
```

Comment:
Is `cumul_cases` here an example of what you meant by:

> **Cumulative totals:** Does front-end want to see a cumulative total to date for each day in time series? …In the cases where daily cumulative numbers are provided, sometimes the totals will not add up, and usually there is a comment somewhere explaining this.

Either way, that’s super interesting. If summing the numbers is not necessarily correct in all cases, then it seems like we should definitely put the cumulative total in there.

Is the comment usually at the level of the whole dataset, or on a given day? Would it go in `meta_from_source` here?

Reply:
It's at the level of the dataset, so the comment would go in meta_from_source.

Reply:
"If summing the numbers is not necessarily correct in all cases, then it seems like we should definitely put the cumulative total in there."  - I think I'm misunderstanding; if summing the numbers is not necessarily correct, wouldn't we not want to put the cumulative total in there? (since the sum isn't correct0

Reply:
I was reacting to this:

> In the cases where daily cumulative numbers are provided, sometimes the totals will not add up, and usually there is a comment somewhere explaining this.

Which is saying, if I’m reading correctly, that some counties *provide* cumulative totals rather than us having to synthesize them by summing the dailies, and that in some cases a synthesized cumulative total wouldn’t match the provided one anyway because of some quirk of the data or because of some other information not surfaced through the API (and usually a description of the discrepancy would be in the metadata).

So based on that, I was thinking that we should include the cumulative total because, in some cases, the cumulative total comes from the county and we couldn’t reproduce it ourselves, even if we downloaded all the data on the front-end.

Does that make sense?